### PR TITLE
Rebase storage tests

### PR DESCRIPTION
**Pull request by troydai**

### CODE REVIEW

File: src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/test_storage.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ def body(self):
         s.cmd('storage account delete -g {} -n {} --force'.format(rg, account))
         s.cmd('storage account check-name --name {}'.format(account), checks=JMESPathCheck('nameAvailable', True))
         result = s.cmd('storage account check-name --name teststorageomega --query "nameAvailable" -o tsv')
-        assert result == 'false'
+        assert result == 'true'
```
Comment (troydai) [inline src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/test_storage.py]: Travis, this is changed in the latest tests. Does the change makes sense to you?

Comment (tjprescott) [inline src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/test_storage.py]: It makes sense if someone created a storage account with that nam... It's supposed to be something unique that isn't used, so we should probably revise the logic here.

Comment (troydai) [inline src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/test_storage.py]: In the same scenario, the account name is checked and ensure to be available at the beginning. There isn't any operations against the 'teststorageomega', therefore I think this change make sense here: it should keep be available.

Comment (troydai) [inline src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/test_storage.py]: If it looks good to you, just approve the PR.

**Pull request merged**